docusaurus.config.js

```diff
@@ -236,6 +247,7 @@ const configuration = {
               { label: 'Overview', to: '/docs/sdks' },
               { label: 'Core SDK', to: '/sdk-docs/core-sdk-ts/overview' },
               { label: 'Wallet SDK', to: '/sdk-docs/wallet-sdk-web/overview' },
+              { label: 'Game engine SDKs', to: '/sdk-docs/unity-sdk/overview' },
```

Comment:
```suggestion
              { label: 'Game Engine SDKs', to: '/sdk-docs/unity-sdk/overview' },
```
Should this be capitalised?

Reply:
Thanks Matt! I think that since "Game engine SDKs" aren't a _type_ of SDK (ie. not a proper name) but rather a category, it shouldn't be capitalised (but see your point about it looking different from the others - but I guess that's because it is)